hip/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 if (GINKGO_HIP_PLATFORM MATCHES "${HIP_PLATFORM_NVIDIA_REGEX}"
-    AND CMAKE_CUDA_COMPILER_VERSION VERSION_LESS 9.2)
+    AND GINKGO_BUILD_CUDA AND CMAKE_CUDA_COMPILER_VERSION VERSION_LESS 9.2)
```

Comment:
could we have method to check nvcc version from hip? it will skip the check when disabling the `GINKGO_BUILD_CUDA`

Reply:
In that case, we already do that:
`find_package(CUDA 9.2 REQUIRED)`